app/src/main/java/com/github/se/travelpouch/AuthenticationService.kt

```diff
@@ -0,0 +1,50 @@
+package com.github.se.travelpouch
+
```

Comment:
We added an adapter to be able to mock the authentication service. So that we can mock it throught an interface it shares with a mock authentication service

Reply:
ok make sense